Editor/Deployment/DeploymentSettings.cs

```diff
@@ -160,12 +170,18 @@ public async Task SetGameNameAsync(string value)
 
         public void Refresh()
         {
-            IEnumerable<DeployerBase> deployers = _scenarioLocator.GetScenarios();
+            var deployers = _scenarioLocator.GetScenarios().ToList();
             _deployers.Clear();
-            _deployers.AddRange(deployers);
-
-            AllScenarios = _deployers.Select(deployer => deployer.DisplayName).ToArray();
+            for (int i = 0; i < deployers.Count(); i++)
+            {
+                var exists = Enum.IsDefined(typeof(DeploymentScenarios), i);
+                if (exists)
+                {
+                    _deployers[(DeploymentScenarios)i] = deployers[i];
+                }
+            }
 
```

Comment:
Is this required if the enum is corrected to read 1,2 and 3? We should cleanup any deployers are that are unused and keep this logic simple.

Reply:
Changed and removed old deployers